src/app/[slug]/page.tsx

```diff
@@ -137,8 +138,7 @@ export async function generateMetadata(
 
 async function getPartner(slug: string, spoofDate?: string) {
   const now = getNow(spoofDate)
-
-  const today = format(now, 'yyyy-MM-dd')
+  const today = getDropDate(spoofDate)
```

Comment:
@lauchness does this rely on spoofdate everywhere?

Reply:
It allows for spoof date, but relys on UTC time under the hood